tests/CustomAttributesTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace TestMonitor\Custify\Tests;
+
+use PHPUnit\Framework\TestCase;
+use TestMonitor\Custify\Resources\CustomAttributes;
+
+class CustomAttributesTest extends TestCase
+{
+    /** @test */
+    public function custom_attributes_should_accept_nummeric_values()
```

Comment:
The test naming deviates from the other tests.

it_should_accept_nummeric_values_when_sending_custom_attributes?